users/models.py

```
@@ -192,6 +192,122 @@ class Badge(models.Model):
+            cls.CHIEF_OF_STAFF.value,
+            cls.CMO.value,
+            cls.CTO.value,
+        )
+
+
+# Sentinel dropdown value for "No Public Role": an explicit opt-out that hides
+# the user's role on every profile surface (see `hide_public_role`). Distinct
+# from an empty selection, which falls through to the auto-derived role.
+NO_PUBLIC_ROLE_OPTION = "__no_public_role__"
+NO_PUBLIC_ROLE_LABEL = (
+    "No Public Role – Your role won't be linked to your name elsewhere on the site."
```

Ruff (0.15.21)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)